packages/i18n/src/resources/en.json

```diff
@@ -260,6 +260,7 @@
   "Move folder hint": "Select a new storage location.",
   "Storage Folder": "Storage Folder",
   "DB_FILE_INVALID": "Invalid Database file",
+  "FILE_ALREADY_EXISTS": "File already exists",
```

Comment:
It might be a good idea to give higher priority to improving the naming conventions for i18n keys, considering the current multitude of approaches.

Reply:
What is the convention btw? `com.affine.xxx`?

Reply:
Let's fix this issue after we got a convention for i18n.